src/main/webapp/features/screens/game/dashboard/cardsPanel/ct-cards-panel.html

```diff
@@ -0,0 +1,38 @@
+<ct-modal-window modal-window-id="CARDS_PANEL">
+
+    <div class="ct-modal-window-bg" ct-modal-window-close></div>
+    <div class="ct-modal-window-content content">
+
+        <button ng-disabled="cards().knight===0">
```

Comment:
we should disable buttons, but show quantity value when it is not user's turn - now it is possible to click on the button if it is not user's turn